canonical-web-and-design/vanilla-framework
# Layouts cleanup

## Done

Fixes #2897 

- extracts common part of HTML layout to one file to share between site and docs layouts

## QA

- Pull code
- Run `./run`
- Open http://0.0.0.0:8101/ or https://vanilla-framework-canonical-web-and-design-pr-2908.run.demo.haus/
- Make sure site works as before
